include/libtorrent/settings_pack.hpp

```diff
@@ -1937,6 +1941,15 @@ namespace aux {
 			// to clamp it in order to allow UDP packets go through
 			dht_max_infohashes_sample_count,
 
+			// this is the minimum allowed announce interval for a WebSocket
+			// tracker used by WebTorrent to signal WebRTC connections. This is
+			// specified in seconds and is used as a sanity check on what is
+			// returned from a tracker.
+			min_websocket_announce_interval,
```

Comment:
Is there a good reason to have a separate setting for this from the normal tracker's min announce interval?

Reply:
There is, this one could legitimately be pretty low (like 30s), plus the fact that the mechanism behind it being different from bittorrent tracker announces (the batch of offers expires), I think it deserves its own setting.